api/src/main/java/bio/terra/tanagra/service/query/api/FilterConverter.java

```diff
@@ -75,11 +79,25 @@ public Filter convert(ApiFilter apiFilter, VariableScope scope) {
         return Filter.BinaryFunction.Operator.EQUALS;
       case LESS_THAN:
         return Filter.BinaryFunction.Operator.LESS_THAN;
+      case DESCENDANT_OF:
+        return Filter.BinaryFunction.Operator.DESCENDANT_OF;
       default:
         throw new BadRequestException("Unknown BinaryFilterOperator: " + apiOperator.toString());
     }
   }
 
+  /** Checks if the operator maybe be used with the attribute or else throws. */
+  private void checkAttributeOperatorMatch(
+      Attribute attribute, Filter.BinaryFunction.Operator operator) {
+    if (operator.equals(Filter.BinaryFunction.Operator.DESCENDANT_OF)
```

Comment:
opt nit: As my own rule of thumb I generally reverse the order of this kind of comparison to offer more null-safety, i.e. CONSTANT.equals(parameter)